include/tgfx/core/Image.h

```diff
@@ -249,24 +249,28 @@ class Image {
   std::shared_ptr<Image> makeOriented(Orientation orientation) const;
 
   /**
-   * Returns a rasterized Image scaled by the specified rasterizationScale. A rasterized Image can
-   * be cached as an independent GPU resource for repeated drawing. By default, an Image directly
-   * backed by an ImageBuffer, an ImageGenerator, or a GPU texture is rasterized. Other image aren’t
-   * rasterized unless implicitly created by this method. For example, if you create a subset Image
-   * from a rasterized Image, the subset Image doesn’t create its own GPU cache but uses the full
-   * resolution cache created by the original Image. If you want the subset Image to create its own
-   * GPU cache, call makeRasterized() on the subset Image. The returned Image always has the same
-   * mipmap state as the original Image.
-   * @param rasterizationScale The factor to scale the Image by when rasterizing. The default value
-   * is 1.0, indicating that the Image should be rasterized at its current size. If the value is
-   * greater than 1.0, it may result in blurring.
-   * @param sampling The sampling options to apply when rasterizing the Image if the
-   * rasterizationScale is not 1.0.
-   * @return If the Image is already rasterized and the rasterizationScale is 1.0, the original
-   * Image is returned. If the rasterizationScale is less than zero, nullptr is returned.
+   * Creates a new Image by scaling this Image to fit the specified width and height. The returned
+   * Image has the same mipmap and rasterization state as the original.
```

Comment:
Creates a new Image scaled to the specified width and height. The new Image keeps the original's mipmap and rasterization settings. If the original Image is rasterized, the scaled Image will also be rasterized and cached at its new size. If not, the scaled Image won't be cached and will only render the needed parts to a temporary offscreen image. To cache the entire scaled Image, use the makeRasterized() method on the returned Image.